src/internal.c

```diff
@@ -15924,6 +15916,187 @@ int DoApplicationData(WOLFSSL* ssl, byte* input, word32* inOutIdx)
     return 0;
 }
 
+const char* AlertTypeToString(int type)
```

Comment:
I took the body of `wolfSSL_alert_type_string_long` and put it in this new `WOLFSSL_LOCAL` function.